### DISCUSSION TITLE
M2 Air Performance seems to have gotten worse over time.

### DISCUSSION THREAD

**Discussion by Eddie-Hartman** [runelite/runelite#17859] [Q&A] [answered]
Hello.

I've noticed while playing over the past year that performance on my M2 air has gotten worse over time. It's OSRS. I understand I'm using an M2 air, but it's wild to me that I'm getting 20 FPS using the same settings killing Lizardmen Shamans that I was doing earlier this year and have probably lost 20 FPS. I'm aware things like zoom really affect it, but I'm wondering if I should really be getting this poor performance or not.

I'm probably in the minority playing on this sort of hardware, but looking for help or tips.

I'm playing stretched with the 117HD plugin. If I zoom all the way out at Lizardman Shamans I get about 18 FPS. Turning on or off 117HD doesn't seem to make a huge difference. I've played with a few settings and the one that had the most impact was the dynamic lighting. Turning that off helped some. Looking at activity monitor I'm GPU bottlenecked.

I'm honestly somewhat more concerned about how I've noticed my FPS going down over time and am wondering what the cause might be or if anyone knows about any enhancements that may be coming. It was definitely fine and playable before, but sometimes I drop so low it's unbearable. I wouldn't think it would take really strong hardware to run this game.

Thank you.

**Comment by Adam-** [comment, marked as answer]
I have an M1 pro, not an M2, but it works okay still for me. Try creating a New Profile, enable the GPU plugin, restart, and see what that does? If it still is slow I would see about making sure the client is running on one of the M2 performance cores and not the efficiency cores.

**Reply by Eddie-Hartman** [reply]
M2 vs M1 Pro would be a big difference. Keep in mind I'm talking about the base that doesn't have a larger GPU. So if you're saying you have an M1 Pro it has a 14 core GPU, while the base M2 in the air will have 8 cores.

I'm not sure how you'd see what core it's running on, but from looking at Activity Monitor I definitely seem more GPU bound than CPU.

I play in full screen and in general the large resolution is what's killing me. Is there a way to set the render resolution while in full screen? I've looked around and searched and couldn't find an option.

Generally speaking, I'm a software dev and fairly technical, if you have something more technical you'd like me to do to help look into it I can. I haven't worked in Java in years though.

Overall, my real concern here is how performance has gotten worse over time, so your suggestion of trying a new profile is definitely a great one I'll definitely do. I can try that another day. Thanks. Just completed Kourend Elite and that was my goal for this weekend lol.

**Reply by Eddie-Hartman** [reply]
I finally circled back to this since I don't play super often. 

It seems entirely a 117HD plugin thing. So the performance there has probably gotten worse over time. I'll switch to just the GPU plugin when doing more intense tasks and try to keep an eye on 117HD. I was really trying to play with 117HD settings to see what could help performance wise, but the only thing I really found was lighting effects, but turning them off was a few FPS improvement and not a major change like I was hoping.

Thanks again for the great profile suggestion.
